perf/NodeJS/Benchmarks.cs

```diff
@@ -9,6 +8,7 @@
 namespace Jering.Javascript.NodeJS.Performance
 {
     [MemoryDiagnoser]
+    [InProcess()]
```

Comment:
Benchmarks are slower on my machine with the `InProcess` attribute (startup is faster but actual benchmark results are ~10ms slower). Is that the case for you too? I think we should aim for the best results since users may compare them to benchmarks for other JS libraries.

Reply:
rolled back changes